src/main/java/seedu/bookmark/ui/help/HelpAdd.java

```diff
@@ -0,0 +1,34 @@
+package seedu.bookmark.ui.help;
+
+public class HelpAdd implements HelpAction {
+
+    private static final String ADD_INTRO = "ADD A BOOK!";
+    private static final String ADD_MESSAGE = "Add books to the storage to keep track of your "
+                                                + "reading progress using our Add Command!";
+    private static final String ADD_COMMAND = "COMMAND: add n/BOOK_TITLE g/GENRE [t/TAG] [t/TAG] tp/TOTAL_PAGES "
```

Comment:
Is there supposed to be two [t/TAG] here to show that users can add multiple tags? If would it be better to explain it in the ADD_NOTES instead?

Reply:
Good Idea! i've made the changes. Thank you :)